.readthedocs.yml

```
@@ -13,6 +13,18 @@ build:
     python: "3"
 
   commands:
+    # https://docs.readthedocs.io/en/stable/build-customization.html#cancel-build-based-on-a-condition
+    #
+    # Cancel building pull requests when there aren't changes in the Doc directory.
+    #
+    # If there are no changes (git diff exits with 0) we force the command to return with 183.
+    # This is a special exit code on Read the Docs that will cancel the build immediately.
+    - if [ "$READTHEDOCS_VERSION_TYPE" = "external" ] && [ "$(git diff --quiet origin/main -- Doc/ .readthedocs.yml; echo $?)" -eq 0 ]; then \
+        echo "No changes to Doc/ - exiting the build."; \
+        exit 183; \
+      fi
```

```suggestion
    - |
      if [ "$READTHEDOCS_VERSION_TYPE" = "external" ] && [ "$(git diff --quiet origin/main -- Doc/ .readthedocs.yml; echo $?)" -eq 0 ];
      then
        echo "No changes to Doc/ - exiting the build.";
        exit 183;
    fi
```

I'm not 100% the indentation is correct, but the idea here is to use `|` in YAML to define it as multi-line and remove all the trailing `\` from the command.

I think it should work this way now that we deployed a fix on RTD.

---

BTW, in my test it works with the trailing `\`. It seems that I only added the `|` at the beginning, but I suppose removing the `\` should also work: https://readthedocs.org/projects/test-builds/builds/20793631/

---

Looks like it's working now, thanks!

Here's more about YAML multilines... https://yaml-multiline.info

I think it's neater like this with `|` and no trailing `\`s.